benchmarks/tools/Build/nuget-local.targets

```
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Project ToolsVersion="15.0">
 
-	<Target Name="ClearPackageCache" BeforeTargets="Build" Condition="'$(CI)' == 'false'">
+	<Target Name="ClearPackageCache" BeforeTargets="DispatchToInnerBuilds" Condition="'$(CI)' == 'false'">
```

Revert benchmark changes
```suggestion
	<Target Name="ClearPackageCache" BeforeTargets="Build" Condition="'$(CI)' == 'false'">
```